CMakeLists.txt

```diff
@@ -344,11 +344,11 @@ set(CMAKE_REQUIRED_INCLUDES ${CURL_INCLUDE_DIRS})
 set(CMAKE_REQUIRED_LIBRARIES ${CURL_LIBRARIES})
 CHECK_SYMBOL_EXISTS(curl_multi_wait curl/curl.h HAVE_CURL_MULTI_WAIT)
 
-find_package(NSS REQUIRED)
-find_package(NSPR REQUIRED)
+find_package(NSS)
```

Comment:
Argh, we still do have some users of NSS in rgw. They are `AES_256_CTR::encrypt` and `rgw_decode_b64_cms`. As the crypto initialization isn't specific to a given daemon, we would need to keep NSS required till rewrite of these things (or rework the init itself and limit that to RGW and tests only). I've missed that, removing the commit. :-/